spec/features/groups/groups_index_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.feature 'Groups Index Page', type: :feature do
```

Comment:
- [ ] @Felipe-Perez-Ferraro As the previous reviewer noted all tests in your spec/features folder are failing. Kindly confirm if it's passing on your end by attaching the screenshot of the test result, below is the result of running your tests
![image](https://github.com/Felipe-Perez-Ferraro/trans_by/assets/61408860/d7180343-bd8c-437d-b8c4-37076a031395)

Reply:
Hello, thank you for your feedback, i've added the tests in the pull request message